### PR TITLE
TST: Work around statsmodels bug

### PR DESCRIPTION
**Pull request by TomAugspurger**
Closes https://github.com/pandas-dev/pandas/issues/24088

### CODE REVIEW

**Comment by TomAugspurger** [comment]
cc @jreback. Merging in a little bit since this is causing CI failures.

**Comment by jreback** [comment]
yep looks good

**Comment by jreback** [comment]
I think this is failing all of the azure builds: ttps://dev.azure.com/pandas-dev/pandas/_build/results?buildId=4678 (I don't have my computer today, otherwise would fix :>)

@datapythonista

File: pandas/tests/plotting/test_datetimelike.py [hunk 1]
```diff
@@ -1557,7 +1557,10 @@ def _check_plot_works(f, freq=None, series=None, *args, **kwargs):
         # GH18439
         # this is supported only in Python 3 pickle since
         # pickle in Python2 doesn't support instancemethod pickling
-        if PY3:
+        # TODO(statsmodels 0.10.0): Remove the statsmodels check
```
Comment (jbrockmendel) [inline pandas/tests/plotting/test_datetimelike.py:1560]: can we remove this yet?

**Pull request merged**